deployment/chart/templates/deployment.yaml

```diff
@@ -50,30 +50,41 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           env:
-          - name: TICKET_DB_HOST
-            value: {{ include "chart.postgresql.fullname" . }}
-          - name: TICKET_DB_PORT
-            value: "5432"
-          - name: TICKET_DB_NAME
-            value: {{ .Values.postgresql.auth.database }}
-          - name: TICKET_DB_USER
-            value: {{ .Values.postgresql.auth.username }}
-          - name: TICKET_DB_PASSWORD
-            value: {{ .Values.postgresql.auth.password }}
+            # These should really come from a autogenerate value
+            - name: TICKET_DB_HOST
+              value: 127.0.0.1
+            - name: TICKET_DB_PORT
+              value: "5432"
+            - name: TICKET_DB_NAME
```

Comment:
The local host only makes sense in development, no?

Reply:
when you have sidecar processes, like the cloudsql proxy container is, you can access it via `localhost` because they are sharing the same "pod user space"